src/view/components/extensionViewForm.jsx

```diff
@@ -7,17 +7,21 @@ import ExtensionViewContext from "./extensionViewContext";
 const ExtensionViewForm = ({
   initialValues,
   getSettings,
-  validate,
-  validationSchema,
+  validateFormikState,
+  formikStateValidationSchema,
+  validateNonFormikState,
   render
 }) => {
   const formikProps = useFormik({
     initialValues,
     enableReinitialize: true,
     onSubmit: () => {},
-    validate: wrapValidateWithErrorLogging(validate),
-    validationSchema,
-    validateOnChange: false
```

Comment:
Notice that I removed `validateOnChange: false`, which means it will not validate on any form state change. I was playing around with disabling it in on the Send Event view, and it was okay there, but it made for some weird user experiences in the XDM object view (e.g., changing a selection within a radio group and the tree still showing that something was invalid until the radio group lost focus). By validating on every change, we're back in alignment with what we're doing on `main`.

Reply:
Sorry, I meant:

> which means it will **now** validate on any form state change

but you probably understood that.